esp/clients/wsdfuaccess/wsdfuaccess.cpp

```diff
@@ -769,9 +769,11 @@ class DFUAccessTests : public CppUnit::TestFixture
     }
 };
 
+unsigned DFUAccessTests::serverPort = DAFILESRV_PORT+1; // do not use standard port, which if in a URL will be converted to local path if IP is local
```

Comment:
This would be probably be clearer if explicitly used MP_START_PORT (which is the start of a reserved dynamic port range) (but in fact it's the same as DAFILESRV_PORT+1
And I'll add a comment explaining why used (for convenience) for unit testing.